redbot/cogs/mod/settings.py

```diff
@@ -59,6 +61,17 @@ async def modset_showsettings(self, ctx: commands.Context):
             if delete_repeats != -1
             else _("No")
         )
+        custom = ""
+        channel_data = await self.config.all_channels()
+        for channel in ctx.guild.channels:
+            data = data = channel_data.get(channel.id, None)
```

Comment:
```suggestion
            data = channel_data.get(channel.id, None)
```
If you aren't setting multiple different variables, there is no reason to use this syntax.